### ISSUE TITLE
[Suggestion] WSADashboard

### ISSUE THREAD

**Issue by DanielGolan-mc** [Simizfo/WSATools#19]
____________________________
|                |
|                |
|.               |
|                |.                                   |
|Open     |_
|apps      |d| main window
|                |  |.                     
-----------------------------|

Now let me explain:
- open apps: apps can be opened in WSATools directly, and those will be shown in here. Only supports one app. Will not show when no app is open.
- d: short for dashboard. Contains a + button, to add apks (you can drag them to the app too), and a three dots button, to open the rest of the dashboard, which will expand upwards. There'll be app settings and ⚙️ button to open wsa settings in the expanded dashboard.
- main windows: shows a list of installed apps, notifications from them, and apps recommended to install.

- optional: the open apps layout will show a launcher app (Poco launcher is a great one) if there's no app attached to the dashboard.

**Comment by Simizfo** [comment]
Hmmm, sorry for that, but I can't see how this can be useful. Right now WSATools is implementing helpful features that aren't available or need several manual steps to be achieved. The end objective is to make things as easy as possible. Normal consumers don't know how to use the ADB CLI to install APKs, so WSATools comes to help with that, as WSA doesn't provide that feature by default. The same comes for Backups. Backups are being implemented because WSA doesn't allow for that natively, and because the normal user doesn't know where and what to copy and replace if they want to do it manually.

Main points of WSATools:
- Good UI (that's why I'm using WinUI2 on UWP)
- Good features that aim to help the most basic of users, everything is designed from their point of view

What your suggestion does, anyway, is already available in some apps I've seen around. Maybe look at them? I don't want to reinvent the wheel, after all, unless I'm going to make it better in both points I've listed up here.

Leaving this open, anyway, I'll rename it and put a discussion label.

**Comment by DanielGolan-mc** [comment]
This suggestion is to make it easier to interact.
The launcher view can, and maybe should, be dropped, but the main window will be a great way to really make it fill more "tools" than "installer that backups".

**Comment by Simizfo** [comment]
Again, sorry, but I still don't see how that can be an improvement. I only have two tools right now on WSATools. I see your proposal more of an "hub" to everything related WSA, which I don't think it fits my original idea for the app. At least, not for now, maybe when it becomes more complex with way more tools.

About supporting me financially: I don't want anything like ads/sponsored apps in this. I hate ads, and I will refrain from putting anything like that in my app for as long as I can. PayPal donations are optional, are hidden in settings (coming in 0.2) and only people who really want to donate will have a way to do so.

**Comment by Simizfo** [comment]
Yes, and I'm going to keep building over it, adding features, improving those that are already here, and so on

But right now the priority keeps being adding useful features that aren't available easily
